rmw_connext_shared_cpp/include/rmw_connext_shared_cpp/event_converter.hpp

```diff
@@ -45,6 +45,6 @@ bool is_event_supported(const rmw_event_type_t & event_t);
   * @return to_return the corresponding rmw_ret_t that maps to the input DDS_ReturnCode_t. By
   * default RMW_RET_ERROR is returned if no corresponding rmw_ret_t is not defined.
   */
-rmw_ret_t check_dds_ret_code(DDS_ReturnCode_t & dds_return_code);
+rmw_ret_t check_dds_ret_code(const DDS_ReturnCode_t & dds_return_code);
```

Comment:
DDS_ReturnCode_t (which should be DDS::ReturnCode_t) is a basic type so why not pass it by value?

Reply:
This has been fixed. Thanks.